### ISSUE TITLE
Video Classification example

### ISSUE THREAD

**Issue by vladimircape** [keras-team/keras-io#1019]
I try video_classification.py
on Tensorflow 2.9
but Test accuracy: 19.64%?

**Comment by sachinprasadhs** [comment]
@sayakpaul , For example guide https://keras.io/examples/vision/video_classification/ link https://git.io/JGc31 seems to be broken, could you please share the updated dataset link, I can go ahead and create a PR. Thanks!

**Comment by sayakpaul** [comment]
https://github.com/sayakpaul/Action-Recognition-in-TensorFlow/releases/download/v1.0.0/ucf101_top5.tar.gz

**Comment by sachinprasadhs** [comment]
Hi @vladimircape ,

Thanks for reporting the issue.

The purpose of the tutorial in general is to demonstrate the workflow of any topic, results are not guaranteed to be the state of the art or benchmark performance.

For you to get better test accuracy, please follow the next steps mentioned in the guide by author.
Note: It may consume significant amount of memory.

**Next steps**
1. In this example, we made use of transfer learning for extracting meaningful features from video frames. You could also fine-tune the pre-trained network to notice how that affects the end results.
2. For speed-accuracy trade-offs, you can try out other models present inside [tf.keras.applications](https://www.tensorflow.org/api_docs/python/tf/keras/applications).
3. Try different combinations of MAX_SEQ_LENGTH to observe how that affects the performance.
4. Train on a higher number of classes and see if you are able to get good performance.
5. Following [this tutorial](https://www.tensorflow.org/hub/tutorials/action_recognition_with_tf_hub), try a [pre-trained action recognition model](https://arxiv.org/abs/1705.07750) from DeepMind.
6. Rolling-averaging can be useful technique for video classification and it can be combined with a standard image classification model to infer on videos. [This tutorial](https://www.pyimagesearch.com/2019/07/15/video-classification-with-keras-and-deep-learning/) will help understand how to use rolling-averaging with an image classifier.
7. When there are variations in between the frames of a video not all the frames might be equally important to decide its category. In those situations, putting a [self-attention layer](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Attention) in the sequence model will likely yield better results.
8. Following [this book chapter](https://livebook.manning.com/book/deep-learning-with-python-second-edition/chapter-11), you can implement Transformers-based models for processing videos.

You can use the trained model hosted on [Hugging Face Hub](https://huggingface.co/keras-io/video-classification-cnn-rnn) and try the demo on [Hugging Face Spaces](https://huggingface.co/spaces/keras-io/keras-video-classification-cnn-rnn).
